SPECS/opensc/opensc.spec

```diff
@@ -41,29 +39,17 @@ supporting this API (such as Mozilla Firefox and Thunderbird) can use it. On
 the card OpenSC implements the PKCS#15 standard and aims to be compatible with
 every software/card that does so, too.
 
-
 %prep
-%setup -q
-%patch1 -p1 -b .pinpad
-
-# The test-pkcs11-tool-allowed-mechanisms already works in Fedora
-sed -i -e '/XFAIL_TESTS/,$ {
-  s/XFAIL_TESTS.*/XFAIL_TESTS=test-pkcs11-tool-test.sh/
-  q
-}' tests/Makefile.am
+%autosetup -p1
 
 cp -p src/pkcs15init/README ./README.pkcs15init
 cp -p src/scconf/README.scconf .
-# No {_libdir} here to avoid multilib conflicts; it's just an example
-sed -i -e 's|/usr/local/towitoko/lib/|/usr/lib/ctapi/|' etc/opensc.conf.example.in
+sed -i -e 's|/usr/local/towitoko/lib/|%{_libdir}/ctapi/|' etc/opensc.conf.example.in
 
 %build
 autoreconf -fvi
-%ifarch %{ix86}
-sed -i -e 's/opensc.conf/opensc-%{_arch}.conf/g' src/libopensc/Makefile.in
-%endif
-sed -i -e 's|"/lib /usr/lib\b|"/%{_lib} %{_libdir}|' configure # lib64 rpaths
```

Comment:
Why don't we need this `sed` anymore?

Reply:
That's specific to Fedora using `/usr/lib64` as their `%{_libdir}`.